src/components/ordered-list-item/OrderedListItem.styles.ts

```diff
@@ -0,0 +1,30 @@
+export const styles = {
+  number: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderColor: 'primary.700',
+    borderRadius: '50%',
+    border: '2px solid',
+    textAlign: 'center',
+    typography: { xs: 'caption', sm: 'subtitle2' },
+    fontWeight: { xs: 500 },
+    minWidth: { xs: '14px', sm: '18px' },
+    height: { xs: '14px', sm: '18px' },
+    lineHeight: { xs: 'initial', sm: 'initial' }
```

Comment:
I guess you can do it like this:
lineHeight: 'initial', because 'xs' is a smaller breakpoint and it works the same for higher breakpoints

Reply:
fixed